spynnaker/pyNN/models/neuron/synaptic_manager.py

```diff
@@ -322,14 +323,20 @@ def get_sdram_usage_in_bytes(
             self, vertex_slice, machine_time_step, application_graph,
             app_vertex):
         in_edges = application_graph.get_edges_ending_at_vertex(app_vertex)
-        return (
-            self._get_synapse_params_size() +
-            self._get_synapse_dynamics_parameter_size(
-                vertex_slice, application_graph, app_vertex) +
-            self._get_synaptic_blocks_size(
-                vertex_slice, in_edges, machine_time_step) +
-            self.__poptable_type.get_master_population_table_size(in_edges) +
-            self._get_size_of_generator_information(in_edges))
+        # These values can be cross checked with the data_spec_text_files
+        # SYNAPSE_PARAMS = 2
+        region_2 = self._get_synapse_params_size()
+        # SYNAPSE_DYNAMICS = 5
+        region_5 = self._get_synapse_dynamics_parameter_size(
+            vertex_slice, application_graph, app_vertex)
+        # SYNAPTIC_MATRIX = 4
+        region_4 = self._get_synaptic_blocks_size(
+            vertex_slice, in_edges, machine_time_step)
+        # POPULATION_TABLE = 3
+        region_3 = self.__poptable_type.get_master_population_table_size(in_edges)
+        #  DIRECT_MATRIX = 10
+        region_10 = self._get_size_of_generator_information(in_edges)
```

Comment:
The generator information goes in region 9.

Reply:
thanks.

Reply:
Ok changed the name to region 9 and move the +1 word to what is now the region4_10 method